rest-api-spec/src/main/resources/rest-api-spec/test/indices.get_settings/10_basic.yml

```diff
@@ -1,173 +1,180 @@
 ---
 setup:
   - do:
-        indices.create:
```

Comment:
You don't need to change the existing yaml test cases, but add a new test case with `skip` section to test the code change only, something like this:
https://github.com/opensearch-project/OpenSearch/blob/f8e886507950d13e7e7d3f7d9bd44241e5ff83b7/modules/ingest-common/src/yamlRestTest/resources/rest-api-spec/test/ingest/340_fingerprint_processor.yml#L10

Reply:
Added a new test case with the `skip` section